src/main/java/core/basesyntax/strategy/Operation.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.strategy;
+
+public interface Operation {
+    int process(int quantity);

```

Comment:
```suggestion
    void process(FruitTransaction transaction);
```
this method should process the operation, it should change a value for that fruit in storage